tox.ini

```diff
@@ -75,6 +75,7 @@ set_env =
     # Workaround for https://github.com/python-poetry/poetry/issues/6958
     POETRY_INSTALLER_PARALLEL = false
     LIBJUJU_VERSION_SPECIFIER = {env:LIBJUJU_VERSION_SPECIFIER:3.2.2}
+    file|microceph.source
```

Comment:
1. it will interfere with other tests with local runs. A simple `touch microceph.source` can fix it though 

2. we use almost all variables. The ones not used are related to ssl enabled ceph, which we need to support eventually

Reply:
Of course, the action can either be improved or turned to a local script for setting up microceph locally, and either using passenv or a file.
Whatever case, i would like to deal with it in a follow up PR.